revup/amend.py

```diff
@@ -44,13 +44,18 @@ async def invoke_editor_for_commit_msg(
         # git's default if the message is being edited (which if we've reached this it is)
         cleanup_type = "strip"
 
+    # Respect the configured comment character
+    comment_ret, comment_char = await git_ctx.git("config", "core.commentChar", raiseonerror=False)
+    if comment_ret != 0:
+        comment_char = "#"
+
     comments = f"{topic_summary}\n{stat_text}"
     if cleanup_type == "scissors":
         comments = f"\n{CLEANUP_SCISSOR_LINE}\n{CLEANUP_SCISSOR_COMMENT}\n{comments}"
     elif cleanup_type == "strip":
-        comments = f"\n{CLEANUP_STRIP_COMMENT}\n{comments}"
+        comments = f"\n{CLEANUP_STRIP_COMMENT.format(comment_char)}\n{comments}"
```

Comment:
Format in an interpolated string is kinda wild, but I can't think of a nicer way lol